src/components/functionBased/TodoContainer.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import TodosList from './TodosList';
+import Header from './Header';
+import InputTodo from './InputTodo';
+
+const getInitialTodos = () => {
+  const temp = localStorage.getItem('todos');
+  const savedTodos = JSON.parse(temp);
+  return { todos: savedTodos } || { todos: [] };
+};
+
+const TodoContainer = () => {
+  const [state, setState] = useState(getInitialTodos());
+  // useEffect(() => {
+  //   const temp = localStorage.getItem('todos');
+  //   const loadedTodos = JSON.parse(temp);
+  //   if (loadedTodos) {
+  //     setState({
+  //       ...state,
+  //       todos: loadedTodos,
+  //     });
+  //   }
+  // }, []);
```

Comment:
It would be better to remove any commented line of codes before mergin to the development branch.

Reply:
These comments were deleted long ago :x:

Reply:
I noticed that in your recent commits yea :v:, when I mentioned this, the code already had those commented lines.